[RELEASE] Release version 2.4.1

## Release OpenSearch and OpenSearch Dashboards 2.4.1

I noticed that a manifest was automatically created in [manifests/2.4.1](/opensearch-project/opensearch-build/tree/main/manifests/2.4.1). Please follow the following checklist to make a release.

<details><summary>How to use this issue</summary>
<p>

## This Release Issue

This issue captures the state of the OpenSearch release, its assignee is responsible for driving the release. Please contact them or @mention them on this issue for help. There are linked issues on components of the release where individual components can be tracked.  More details are included in the Maintainers [Release owner](https://github.com/opensearch-project/opensearch-build/blob/main/MAINTAINERS.md#release-owner) section.

## Release Steps

There are several steps to the release process, these steps are completed as the whole release and components that are behind present risk to the release.  The release owner completes the tasks in this ticket, whereas component owners resolve tasks on their ticket in their repositories.

Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.

### Component List

To aid in understanding the state of the release there is a table with status indicating each component state. This is updated based on the status of the component issues.

</p>
</details>

### Preparation

- [x] Assign this issue to a release owner.
- [x] Declare a pencils down date for new features to be merged. 
- [x] __Dec 07, 2022__ is pencils down date for feature freeze.
- [ ] ~Update the Campaigns section to include monitoring campaigns during this release.~
- [x] Update this issue so all `__REPLACE_RELEASE-__` placeholders have actual dates.
- [ ] Document any new quality requirements or changes.
- [x] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1). 
- [ ] ~[Create a release issue in every component repo](https://github.com/opensearch-project/opensearch-build/blob/main/meta/README.md#create-a-release-issue) that links back to this issue, update Components section with these links.~
- [ ] Ensure the label is created in each component repo for this new version, and the next minor release. [Create a version label](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-or-update-labels-in-all-plugin-repos)
- [ ] ~Ensure that all release issues created above are assigned to an owner in the component team.~
- [x] Increase the build frequency for the this release from once a day (H 1 * * *) to once every hour (H/60 * * * *) in [jenkinsFile](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/check-for-build.jenkinsfile)

### CI/CD (Feature Freeze) - _Ends __Dec 07, 2022__

- [x] Create Jenkins workflows that run daily snapshot builds for OpenSearch and OpenSearch Dashboards. 
- [x] Increment each component version to 2.4.1 and ensure working CI in component repositories.
- [x] Make pull requests to add each component to [manifests/2.4.1/opensearch-2.4.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.4.1/opensearch-2.4.1.yml) and [manifests/2.4.1/opensearch-dashboards-2.4.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.4.1/opensearch-dashboards-2.4.1.yml) with the corresponding checks.
- [x] OpenSearch / OpenSearch-Dashboards core and components teams finalize their features
- [x] OpenSearch / OpenSearch-Dashboards core cut branch `<MajorVersion>.<MinorVersion>` early.

### Campaigns

__REPLACE with OpenSearch wide initiatives to improve quality and consistency.__

### Code Complete - _Ends __Dec 07, 2022___

- [x] Code Complete: Make sure that the code for this specific version of the release is ready and the branch corresponding to this release has been added to this release version manifest.
- [x] Verify pull requests to add each component to [manifests/2.4.1/opensearch-2.4.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.4.1/opensearch-2.4.1.yml) and [manifests/2.4.1/opensearch-dashboards-2.4.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.4.1/opensearch-dashboards-2.4.1.yml) have been merged.
- [x] Gather, review and combine the release notes from components repositories.

### Release testing - _Ends __Dec 12, 2022__

- [x] All components should have cut branch `<MajorVersion>.<MinorVersion>` for the release.
- [x] Declare a release candidate build, and provide the instructions with the release candidates for teams on testing (__REPLACE_RELEASE-minus-8-days__).
- [x] Stop builds for this version of OpenSearch and/or OpenSearch Dashboards in order to avoid accidental commits going in unknowingly. Restart only if necessary else manually run the build workflow and declare new release candidate.
- [x] Sanity Testing (__REPLACE_RELEASE-minus-8-days__ - __REPLACE_RELEASE-minus-6-days__): Sanity testing and fixing of critical issues found by teams. Teams test their components within the distribution, ensuring integration, backwards compatibility, and perf tests pass.
- [x] Publish all test results in the comments of this issue.

### Performance testing validation - _Ends __REPLACE_RELEASE-minus-6-days___

- [ ] ~Performance tests do not show a regression~
- [ ] ~Longevity tests do not show any issues~

### Release - _Ends Dec 13, 2022_

- [ ] Verify [all issues labeled `v2.4.1` in all projects](https://github.com/opensearch-project/project-meta#find-labeled-issues) have been resolved.
- [x] Complete [documentation](https://github.com/opensearch-project/documentation-website) for this release.
- [x] Author [blog post](https://github.com/opensearch-project/project-website) for this release.
- [x] __Dec 12,2022__ - Publish this release on [opensearch.org](https://opensearch.org/downloads.html).
- [x] __Dec 13, 2022__ - Publish a [blog post](https://github.com/opensearch-project/project-website) - release is launched!

### Post Release

- [x] Create [release tags](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/release-tag/release-tag.jenkinsfile) for each component (Jenkins job name: release-tag-creation).
- [x] Replace refs in [manifests/2.4.1](/opensearch-project/opensearch-build/tree/main/manifests/2.4.1) with tags and remove checks.
- [x] If this is a major or minor version release, stop building previous patch version.
- [x] Generate distribution release notes reviewed by PM team for opensearch-build repository.
- [x] Increment version for Helm Charts [(sample PR)](https://github.com/opensearch-project/helm-charts/pull/246) for the `2.4.1` release.
- [x] Increment version for Ansible Charts [(sample PR)](https://github.com/opensearch-project/ansible-playbook/pull/50) for the `2.4.1` release.
- [x] Prepare [for next patch release](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#increment-a-version-in-every-plugin) by incrementing patch versions for each component.
- [x] Update [this template](https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md) with any new or missed steps.
- [x] #2981

### Components

__Replace with links to all component tracking issues.__

| Component | On track | Release Notes |
| --------- | -------- | ----- |
| {COMPONENT_ISSUE_LINK} | {INDICATOR}} | {STATUS} |

<details><summary>Legend</summary>
<p>

| Symbol | Meaning |
| -------- | ---------- |
| :green_circle: | On track with overall release |
| :yellow_circle: | Missed last milestone |
| :red_circle: | Missed multiple milestones |

</p>
</details>

Also windows integ test failing for **asynchronous-search** without security:
```
2022-12-08 22:33:41 INFO     ===============================================
2022-12-08 22:33:41 INFO     Running integration tests for asynchronous-search
2022-12-08 22:33:41 INFO     ===============================================
2022-12-08 22:33:41 INFO     Executing "bash C:\Users\Administrator\opensearch-build\scripts\default\integtest.sh -b localhost -p 9200 -s false -v 2.4.1" in C:\Users\ADMINI~1\AppData\Local\Temp\2\tmppcxig_tl\asynchronous-search
2022-12-08 22:37:36 INFO     Recording component test results for asynchronous-search at C:\Users\Administrator\opensearch-build\test-results\3456\integ-test\asynchronous-search\without-security\test-results
2022-12-08 22:37:36 INFO     Integration test run failed for component asynchronous-search
2022-12-08 22:37:36 INFO     OpenJDK 64-Bit Server VM warning: Ignoring option --illegal-access=warn; support was removed in 17.0

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.search.asynchronous.restIT.AsynchronousSearchRestIT.testGetWithKeepAliveUpdate" -Dtests.seed=EBCF501A714D591A -Dtests.security.manager=false -Dtests.locale=hi-IN -Dtests.timezone=Asia/Vientiane -Druntime.java=17


Suite: Test class org.opensearch.search.asynchronous.restIT.AsynchronousSearchRestIT
  2> ?????? ??, ???? ?:??:?? ????????? org.opensearch.client.RestClient logResponse
  2> WARNING: request [POST http://localhost:9200/_opendistro/_asynchronous_search?keep_on_completion=true] returned 1 warnings: [299 OpenSearch-2.4.1-f2f809ea280ffba217451da894a5899f1cec02ab "[POST /_opendistro/_asynchronous_search] is deprecated! Use [POST /_plugins/_asynchronous_search] instead."]
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.search.asynchronous.restIT.AsynchronousSearchRestIT.testGetWithKeepAliveUpdate" -Dtests.seed=EBCF501A714D591A -Dtests.security.manager=false -Dtests.locale=hi-IN -Dtests.timezone=Asia/Vientiane -Druntime.java=17
  2> java.lang.AssertionError:
    Expected: a value greater than <1670557020650L>
         but: <1670557020650L> was equal to <1670557020650L>
        at __randomizedtesting.SeedInfo.seed([EBCF501A714D591A:A08B2D04250F4F1]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:964)
        at org.junit.Assert.assertThat(Assert.java:930)
        at org.opensearch.search.asynchronous.restIT.AsynchronousSearchRestIT.testGetWithKeepAliveUpdate(AsynchronousSearchRestIT.java:135)
  2> NOTE: leaving temporary files on disk at: C:\Users\Administrator\AppData\Local\Temp\2\tmppcxig_tl\asynchronous-search\build\testrun\integTest\temp\org.opensearch.search.asynchronous.restIT.AsynchronousSearchRestIT_EBCF501A714D591A-001
  2> NOTE: test params are: codec=Asserting(Lucene94), sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=hi-IN, timezone=Asia/Vientiane
  2> NOTE: Windows Server 2019 10.0 amd64/Eclipse Adoptium 17.0.5 (64-bit)/cpus=16,threads=1,free=372051760,total=536870912
  2> NOTE: All tests run in this JVM: [AsynchronousSearchQueryIT, AsynchronousSearchRejectionIT, AsynchronousSearchStatsIT, AsynchronousSearchTaskCancellationIT, DeleteAsynchronousSearchSingleNodeIT, GetAsynchronousSearchSingleNodeIT, MixedOperationSingleNodeIT, SubmitAsynchronousSearchSingleNodeIT, AsynchronousSearchCancellationIT, AsynchronousSearchPartialResponseIT, AsynchronousSearchProgressListenerIT, AsynchronousSearchManagementServiceIT, AsynchronousSearchRequestRoutingIT, ApiParamsValidationIT, AsynchronousSearchRestIT]

75 tests completed, 1 failed

=== Standard output of node `node{::integTest-0}` ===

=== Standard error of node `node{::integTest-0}` ===

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':integTest'.
> There were failing tests. See the report at: file:///C:/Users/Administrator/AppData/Local/Temp/2/tmppcxig_tl/asynchronous-search/build/reports/tests/integTest/index.html

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3m 54s

2022-12-08 22:37:36 INFO     Sending SIGKILL to PID 7992
2022-12-08 22:37:36 INFO     Process killed with exit code None
2022-12-08 22:37:36 INFO     Cleanup C:\Users\ADMINI~1\AppData\Local\Temp\2\tmppcxig_tl\1\local-test-cluster/* content after the test
2022-12-08 22:37:37 INFO     Recording local cluster logs for asynchronous-search with test configuration as without-security at C:\Users\Administrator\opensearch-build\test-results\3456\integ-test\asynchronous-search\without-security\local-cluster-logs
2022-12-08 22:37:37 INFO     Skipping sql
2022-12-08 22:37:37 INFO     Skipping notifications-core
2022-12-08 22:37:37 INFO     Skipping notifications
2022-12-08 22:37:37 INFO     Skipping alerting
2022-12-08 22:37:37 INFO     Skipping index-management
2022-12-08 22:37:37 INFO     Skipping security-analytics
2022-12-08 22:37:37 INFO     Removing C:\Users\ADMINI~1\AppData\Local\Temp\2\tmppcxig_tl
2022-12-08 22:37:37 WARNING  Removing try count: 1/3 for C:\Users\ADMINI~1\AppData\Local\Temp\2\tmppcxig_tl\asynchronous-search\.git\objects\pack\pack-c4ebb968f4579a7108e230120ea5c4bce2d8acac.idx
2022-12-08 22:37:37 WARNING  Removing try count: 1/3 for C:\Users\ADMINI~1\AppData\Local\Temp\2\tmppcxig_tl\asynchronous-search\.git\objects\pack\pack-c4ebb968f4579a7108e230120ea5c4bce2d8acac.pack
2022-12-08 22:37:37 INFO     | asynchronous-search  | with-security        | PASS  |    0 |
2022-12-08 22:37:37 ERROR    | asynchronous-search  | without-security     | FAIL  |    1 |
C:\Users\Administrator\.virtualenvs\opensearch-build-ppYgApDN\lib\site-packages\urllib3\connectionpool.py:1052: InsecureRequestWarning: Unverified HTTPS request is being made to host 'localhost'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
C:\Users\Administrator\.virtualenvs\opensearch-build-ppYgApDN\lib\site-packages\urllib3\connectionpool.py:1052: InsecureRequestWarning: Unverified HTTPS request is being made to host 'localhost'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,

```

Notifications is failing for **windows** too:
```
2022-12-08 22:45:33 INFO     ===============================================
2022-12-08 22:45:33 INFO     Running integration tests for notifications
2022-12-08 22:45:33 INFO     ===============================================
2022-12-08 22:45:33 INFO     Executing "bash C:\Users\Administrator\opensearch-build\scripts\default\integtest.sh -b localhost -p 9200 -s true -v 2.4.1" in C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpyfl4acc0\notifications\notifications
2022-12-08 22:49:58 INFO     Recording component test results for notifications at C:\Users\Administrator\opensearch-build\test-results\3455\integ-test\notifications\with-security\test-results
2022-12-08 22:49:58 INFO     Integration test run failed for component notifications
2022-12-08 22:49:58 INFO     OpenJDK 64-Bit Server VM warning: Ignoring option --illegal-access=warn; support was removed in 17.0

REPRODUCE WITH: gradlew ':notifications:integTest' --tests "org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete some items from multiple notification config with missing configs should fail" -Dtests.seed=E1F5A8212A0E06CE -Dtests.security.manager=false -Dtests.locale=nb -Dtests.timezone=America/Cayenne -Druntime.java=17


Suite: Test class org.opensearch.integtest.config.DeleteNotificationConfigIT
  2> REPRODUCE WITH: gradlew ':notifications:integTest' --tests "org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete some items from multiple notification config with missing configs should fail" -Dtests.seed=E1F5A8212A0E06CE -Dtests.security.manager=false -Dtests.locale=nb -Dtests.timezone=America/Cayenne -Druntime.java=17
  2> java.lang.AssertionError: expected:<19> but was:<18>
        at __randomizedtesting.SeedInfo.seed([E1F5A8212A0E06CE:EDCD62314813E807]:0)
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotEquals(Assert.java:835)
        at org.junit.Assert.assertEquals(Assert.java:120)
        at org.junit.Assert.assertEquals(Assert.java:146)
        at org.opensearch.notifications.ObjectEqualsHelpersKt.verifyMultiConfigIdEquals(ObjectEqualsHelpers.kt:156)
        at org.opensearch.notifications.ObjectEqualsHelpersKt.verifyMultiConfigIdEquals$default(ObjectEqualsHelpers.kt:151)
        at org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete some items from multiple notification config with missing configs should fail(DeleteNotificationConfigIT.kt:159)
  2> NOTE: leaving temporary files on disk at: C:\Users\Administrator\AppData\Local\Temp\2\tmpyfl4acc0\notifications\notifications\notifications\build\testrun\integTest\temp\org.opensearch.integtest.config.DeleteNotificationConfigIT_E1F5A8212A0E06CE-001
  2> NOTE: test params are: codec=Asserting(Lucene94), sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=nb, timezone=America/Cayenne
  2> NOTE: Windows Server 2019 10.0 amd64/Eclipse Adoptium 17.0.5 (64-bit)/cpus=16,threads=1,free=255396024,total=536870912
  2> NOTE: All tests run in this JVM: [NotificationsPluginIT, SecurityNotificationIT, ChimeNotificationConfigCrudIT, CreateNotificationConfigIT, DeleteNotificationConfigIT]

75 tests completed, 1 failed

=== Standard output of node `node{:notifications:integTest-0}` ===

=== Standard error of node `node{:notifications:integTest-0}` ===

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':notifications:integTest'.
> There were failing tests. See the report at: file:///C:/Users/Administrator/AppData/Local/Temp/2/tmpyfl4acc0/notifications/notifications/notifications/build/reports/tests/integTest/index.html

* Try:
> Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':notifications:integTest'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:147)
        at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:282)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:145)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:133)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:74)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:333)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:320)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:313)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:299)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.lambda$run$0(DefaultPlanExecutor.java:143)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:227)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:218)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:140)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
Caused by: org.gradle.api.tasks.VerificationException: There were failing tests. See the report at: file:///C:/Users/Administrator/AppData/Local/Temp/2/tmpyfl4acc0/notifications/notifications/notifications/build/reports/tests/integTest/index.html
        at org.gradle.api.tasks.testing.AbstractTestTask.handleTestFailures(AbstractTestTask.java:651)
        at org.gradle.api.tasks.testing.AbstractTestTask.handleCollectedResults(AbstractTestTask.java:513)
        at org.gradle.api.tasks.testing.AbstractTestTask.executeTests(AbstractTestTask.java:508)
        at org.gradle.api.tasks.testing.Test.executeTests(Test.java:689)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:58)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:51)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:29)
        at org.gradle.api.internal.tasks.execution.TaskExecution$3.run(TaskExecution.java:242)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:68)
        at org.gradle.api.internal.tasks.execution.TaskExecution.executeAction(TaskExecution.java:227)
        at org.gradle.api.internal.tasks.execution.TaskExecution.executeActions(TaskExecution.java:210)
        at org.gradle.api.internal.tasks.execution.TaskExecution.executeWithPreviousOutputFiles(TaskExecution.java:193)
        at org.gradle.api.internal.tasks.execution.TaskExecution.execute(TaskExecution.java:171)
        at org.gradle.internal.execution.steps.ExecuteStep.executeInternal(ExecuteStep.java:89)
        at org.gradle.internal.execution.steps.ExecuteStep.access$000(ExecuteStep.java:40)
        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:53)
        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:50)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:50)
        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:40)
        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:68)
        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:38)
        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:48)
        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:36)
        at org.gradle.internal.execution.steps.CancelExecutionStep.execute(CancelExecutionStep.java:41)
        at org.gradle.internal.execution.steps.TimeoutStep.executeWithoutTimeout(TimeoutStep.java:74)
        at org.gradle.internal.execution.steps.TimeoutStep.execute(TimeoutStep.java:55)
        at org.gradle.internal.execution.steps.CreateOutputsStep.execute(CreateOutputsStep.java:51)
        at org.gradle.internal.execution.steps.CreateOutputsStep.execute(CreateOutputsStep.java:29)
        at org.gradle.internal.execution.steps.CaptureStateAfterExecutionStep.execute(CaptureStateAfterExecutionStep.java:61)
        at org.gradle.internal.execution.steps.CaptureStateAfterExecutionStep.execute(CaptureStateAfterExecutionStep.java:42)
        at org.gradle.internal.execution.steps.BroadcastChangingOutputsStep.execute(BroadcastChangingOutputsStep.java:60)
        at org.gradle.internal.execution.steps.BroadcastChangingOutputsStep.execute(BroadcastChangingOutputsStep.java:27)
        at org.gradle.internal.execution.steps.BuildCacheStep.executeWithoutCache(BuildCacheStep.java:180)
        at org.gradle.internal.execution.steps.BuildCacheStep.lambda$execute$1(BuildCacheStep.java:75)
        at org.gradle.internal.Either$Right.fold(Either.java:175)
        at org.gradle.internal.execution.caching.CachingState.fold(CachingState.java:59)
        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:73)
        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:48)
        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:36)
        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:25)
        at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:36)
        at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:22)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.executeBecause(SkipUpToDateStep.java:110)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.lambda$execute$2(SkipUpToDateStep.java:56)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:56)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:38)
        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:73)
        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:44)
        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:37)
        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:27)
        at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:89)
        at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:50)
        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:114)
        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:57)
        at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:76)
        at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:50)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.executeWithNoEmptySources(SkipEmptyWorkStep.java:249)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.executeWithNoEmptySources(SkipEmptyWorkStep.java:204)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.execute(SkipEmptyWorkStep.java:83)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.execute(SkipEmptyWorkStep.java:54)
        at org.gradle.internal.execution.steps.RemoveUntrackedExecutionStateStep.execute(RemoveUntrackedExecutionStateStep.java:32)
        at org.gradle.internal.execution.steps.RemoveUntrackedExecutionStateStep.execute(RemoveUntrackedExecutionStateStep.java:21)
        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsStartedStep.execute(MarkSnapshottingInputsStartedStep.java:38)
        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:43)
        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:31)
        at org.gradle.internal.execution.steps.AssignWorkspaceStep.lambda$execute$0(AssignWorkspaceStep.java:40)
        at org.gradle.api.internal.tasks.execution.TaskExecution$4.withWorkspace(TaskExecution.java:287)
        at org.gradle.internal.execution.steps.AssignWorkspaceStep.execute(AssignWorkspaceStep.java:40)
        at org.gradle.internal.execution.steps.AssignWorkspaceStep.execute(AssignWorkspaceStep.java:30)
        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:37)
        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:27)
        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:44)
        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:33)
        at org.gradle.internal.execution.impl.DefaultExecutionEngine$1.execute(DefaultExecutionEngine.java:76)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:144)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:133)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:74)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:333)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:320)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:313)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:299)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.lambda$run$0(DefaultPlanExecutor.java:143)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:227)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:218)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:140)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)


* Get more help at https://help.gradle.org

BUILD FAILED in 4m 24s

````

Notifications is failing for both with and without security for windows as well:
```
```
2022-12-08 22:50:29 INFO     ===============================================
2022-12-08 22:50:29 INFO     Running integration tests for notifications
2022-12-08 22:50:29 INFO     ===============================================
2022-12-08 22:50:29 INFO     Executing "bash C:\Users\Administrator\opensearch-build\scripts\default\integtest.sh -b localhost -p 9200 -s false -v 2.4.1" in C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpyfl4acc0\notifications\notifications
2022-12-08 22:53:01 INFO     Recording component test results for notifications at C:\Users\Administrator\opensearch-build\test-results\3455\integ-test\notifications\without-security\test-results
2022-12-08 22:53:01 INFO     Integration test run failed for component notifications
2022-12-08 22:53:01 INFO     OpenJDK 64-Bit Server VM warning: Ignoring option --illegal-access=warn; support was removed in 17.0

REPRODUCE WITH: gradlew ':notifications:integTest' --tests "org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete some items from multiple notification config with missing configs should fail" -Dtests.seed=F363E8B88C588A4C -Dtests.security.manager=false -Dtests.locale=en -Dtests.timezone=Europe/Athens -Druntime.java=17

REPRODUCE WITH: gradlew ':notifications:integTest' --tests "org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete partial items from multiple notification config" -Dtests.seed=F363E8B88C588A4C -Dtests.security.manager=false -Dtests.locale=en -Dtests.timezone=Europe/Athens -Druntime.java=17


Suite: Test class org.opensearch.integtest.config.DeleteNotificationConfigIT
  2> REPRODUCE WITH: gradlew ':notifications:integTest' --tests "org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete some items from multiple notification config with missing configs should fail" -Dtests.seed=F363E8B88C588A4C -Dtests.security.manager=false -Dtests.locale=en -Dtests.timezone=Europe/Athens -Druntime.java=17
  2> java.lang.AssertionError: expected:<19> but was:<18>
        at __randomizedtesting.SeedInfo.seed([F363E8B88C588A4C:FF5B22A8EE456485]:0)
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotEquals(Assert.java:835)
        at org.junit.Assert.assertEquals(Assert.java:120)
        at org.junit.Assert.assertEquals(Assert.java:146)
        at org.opensearch.notifications.ObjectEqualsHelpersKt.verifyMultiConfigIdEquals(ObjectEqualsHelpers.kt:156)
        at org.opensearch.notifications.ObjectEqualsHelpersKt.verifyMultiConfigIdEquals$default(ObjectEqualsHelpers.kt:151)
        at org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete some items from multiple notification config with missing configs should fail(DeleteNotificationConfigIT.kt:159)
  2> REPRODUCE WITH: gradlew ':notifications:integTest' --tests "org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete partial items from multiple notification config" -Dtests.seed=F363E8B88C588A4C -Dtests.security.manager=false -Dtests.locale=en -Dtests.timezone=Europe/Athens -Druntime.java=17
  2> java.lang.AssertionError: expected:<19> but was:<18>
        at __randomizedtesting.SeedInfo.seed([F363E8B88C588A4C:81F90FA9EE09F3FE]:0)
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotEquals(Assert.java:835)
        at org.junit.Assert.assertEquals(Assert.java:120)
        at org.junit.Assert.assertEquals(Assert.java:146)
        at org.opensearch.notifications.ObjectEqualsHelpersKt.verifyMultiConfigIdEquals(ObjectEqualsHelpers.kt:156)
        at org.opensearch.notifications.ObjectEqualsHelpersKt.verifyMultiConfigIdEquals$default(ObjectEqualsHelpers.kt:151)
        at org.opensearch.integtest.config.DeleteNotificationConfigIT.test Delete partial items from multiple notification config(DeleteNotificationConfigIT.kt:197)
  2> NOTE: leaving temporary files on disk at: C:\Users\Administrator\AppData\Local\Temp\2\tmpyfl4acc0\notifications\notifications\notifications\build\testrun\integTest\temp\org.opensearch.integtest.config.DeleteNotificationConfigIT_F363E8B88C588A4C-001
  2> NOTE: test params are: codec=Asserting(Lucene94): {}, docValues:{}, maxPointsInLeafNode=476, maxMBSortInHeap=7.129175445289429, sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=en, timezone=Europe/Athens
  2> NOTE: Windows Server 2019 10.0 amd64/Eclipse Adoptium 17.0.5 (64-bit)/cpus=16,threads=1,free=310048192,total=536870912
  2> NOTE: All tests run in this JVM: [DeleteNotificationConfigIT]

61 tests completed, 2 failed

=== Standard output of node `node{:notifications:integTest-0}` ===

=== Standard error of node `node{:notifications:integTest-0}` ===

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':notifications:integTest'.
> There were failing tests. See the report at: file:///C:/Users/Administrator/AppData/Local/Temp/2/tmpyfl4acc0/notifications/notifications/notifications/build/reports/tests/integTest/index.html

* Try:
> Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':notifications:integTest'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:147)
        at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:282)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:145)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:133)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:74)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:333)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:320)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:313)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:299)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.lambda$run$0(DefaultPlanExecutor.java:143)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:227)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:218)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:140)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
Caused by: org.gradle.api.tasks.VerificationException: There were failing tests. See the report at: file:///C:/Users/Administrator/AppData/Local/Temp/2/tmpyfl4acc0/notifications/notifications/notifications/build/reports/tests/integTest/index.html
        at org.gradle.api.tasks.testing.AbstractTestTask.handleTestFailures(AbstractTestTask.java:651)
        at org.gradle.api.tasks.testing.AbstractTestTask.handleCollectedResults(AbstractTestTask.java:513)
        at org.gradle.api.tasks.testing.AbstractTestTask.executeTests(AbstractTestTask.java:508)
        at org.gradle.api.tasks.testing.Test.executeTests(Test.java:689)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:58)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:51)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:29)
        at org.gradle.api.internal.tasks.execution.TaskExecution$3.run(TaskExecution.java:242)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:68)
        at org.gradle.api.internal.tasks.execution.TaskExecution.executeAction(TaskExecution.java:227)
        at org.gradle.api.internal.tasks.execution.TaskExecution.executeActions(TaskExecution.java:210)
        at org.gradle.api.internal.tasks.execution.TaskExecution.executeWithPreviousOutputFiles(TaskExecution.java:193)
        at org.gradle.api.internal.tasks.execution.TaskExecution.execute(TaskExecution.java:171)
        at org.gradle.internal.execution.steps.ExecuteStep.executeInternal(ExecuteStep.java:89)
        at org.gradle.internal.execution.steps.ExecuteStep.access$000(ExecuteStep.java:40)
        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:53)
        at org.gradle.internal.execution.steps.ExecuteStep$1.call(ExecuteStep.java:50)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:50)
        at org.gradle.internal.execution.steps.ExecuteStep.execute(ExecuteStep.java:40)
        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:68)
        at org.gradle.internal.execution.steps.RemovePreviousOutputsStep.execute(RemovePreviousOutputsStep.java:38)
        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:48)
        at org.gradle.internal.execution.steps.ResolveInputChangesStep.execute(ResolveInputChangesStep.java:36)
        at org.gradle.internal.execution.steps.CancelExecutionStep.execute(CancelExecutionStep.java:41)
        at org.gradle.internal.execution.steps.TimeoutStep.executeWithoutTimeout(TimeoutStep.java:74)
        at org.gradle.internal.execution.steps.TimeoutStep.execute(TimeoutStep.java:55)
        at org.gradle.internal.execution.steps.CreateOutputsStep.execute(CreateOutputsStep.java:51)
        at org.gradle.internal.execution.steps.CreateOutputsStep.execute(CreateOutputsStep.java:29)
        at org.gradle.internal.execution.steps.CaptureStateAfterExecutionStep.execute(CaptureStateAfterExecutionStep.java:61)
        at org.gradle.internal.execution.steps.CaptureStateAfterExecutionStep.execute(CaptureStateAfterExecutionStep.java:42)
        at org.gradle.internal.execution.steps.BroadcastChangingOutputsStep.execute(BroadcastChangingOutputsStep.java:60)
        at org.gradle.internal.execution.steps.BroadcastChangingOutputsStep.execute(BroadcastChangingOutputsStep.java:27)
        at org.gradle.internal.execution.steps.BuildCacheStep.executeWithoutCache(BuildCacheStep.java:180)
        at org.gradle.internal.execution.steps.BuildCacheStep.lambda$execute$1(BuildCacheStep.java:75)
        at org.gradle.internal.Either$Right.fold(Either.java:175)
        at org.gradle.internal.execution.caching.CachingState.fold(CachingState.java:59)
        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:73)
        at org.gradle.internal.execution.steps.BuildCacheStep.execute(BuildCacheStep.java:48)
        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:36)
        at org.gradle.internal.execution.steps.StoreExecutionStateStep.execute(StoreExecutionStateStep.java:25)
        at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:36)
        at org.gradle.internal.execution.steps.RecordOutputsStep.execute(RecordOutputsStep.java:22)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.executeBecause(SkipUpToDateStep.java:110)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.lambda$execute$2(SkipUpToDateStep.java:56)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:56)
        at org.gradle.internal.execution.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:38)
        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:73)
        at org.gradle.internal.execution.steps.ResolveChangesStep.execute(ResolveChangesStep.java:44)
        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:37)
        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsFinishedStep.execute(MarkSnapshottingInputsFinishedStep.java:27)
        at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:89)
        at org.gradle.internal.execution.steps.ResolveCachingStateStep.execute(ResolveCachingStateStep.java:50)
        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:114)
        at org.gradle.internal.execution.steps.ValidateStep.execute(ValidateStep.java:57)
        at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:76)
        at org.gradle.internal.execution.steps.CaptureStateBeforeExecutionStep.execute(CaptureStateBeforeExecutionStep.java:50)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.executeWithNoEmptySources(SkipEmptyWorkStep.java:249)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.executeWithNoEmptySources(SkipEmptyWorkStep.java:204)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.execute(SkipEmptyWorkStep.java:83)
        at org.gradle.internal.execution.steps.SkipEmptyWorkStep.execute(SkipEmptyWorkStep.java:54)
        at org.gradle.internal.execution.steps.RemoveUntrackedExecutionStateStep.execute(RemoveUntrackedExecutionStateStep.java:32)
        at org.gradle.internal.execution.steps.RemoveUntrackedExecutionStateStep.execute(RemoveUntrackedExecutionStateStep.java:21)
        at org.gradle.internal.execution.steps.legacy.MarkSnapshottingInputsStartedStep.execute(MarkSnapshottingInputsStartedStep.java:38)
        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:43)
        at org.gradle.internal.execution.steps.LoadPreviousExecutionStateStep.execute(LoadPreviousExecutionStateStep.java:31)
        at org.gradle.internal.execution.steps.AssignWorkspaceStep.lambda$execute$0(AssignWorkspaceStep.java:40)
        at org.gradle.api.internal.tasks.execution.TaskExecution$4.withWorkspace(TaskExecution.java:287)
        at org.gradle.internal.execution.steps.AssignWorkspaceStep.execute(AssignWorkspaceStep.java:40)
        at org.gradle.internal.execution.steps.AssignWorkspaceStep.execute(AssignWorkspaceStep.java:30)
        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:37)
        at org.gradle.internal.execution.steps.IdentityCacheStep.execute(IdentityCacheStep.java:27)
        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:44)
        at org.gradle.internal.execution.steps.IdentifyStep.execute(IdentifyStep.java:33)
        at org.gradle.internal.execution.impl.DefaultExecutionEngine$1.execute(DefaultExecutionEngine.java:76)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:144)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:133)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:74)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:333)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:320)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:313)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:299)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.lambda$run$0(DefaultPlanExecutor.java:143)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:227)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:218)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:140)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)


* Get more help at https://help.gradle.org

BUILD FAILED in 2m 30s

2022-12-08 22:53:01 INFO     Sending SIGKILL to PID 9788
2022-12-08 22:53:01 INFO     Process killed with exit code None
2022-12-08 22:53:01 INFO     Cleanup C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpyfl4acc0\1\local-test-cluster/* content after the test
2022-12-08 22:53:02 INFO     Recording local cluster logs for notifications with test configuration as without-security at C:\Users\Administrator\opensearch-build\test-results\3455\integ-test\notifications\without-security\local-cluster-logs
2022-12-08 22:53:02 INFO     Skipping alerting
2022-12-08 22:53:02 INFO     Skipping index-management
2022-12-08 22:53:02 INFO     Skipping security-analytics
2022-12-08 22:53:02 INFO     Removing C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpyfl4acc0
2022-12-08 22:53:02 WARNING  Removing try count: 1/3 for C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpyfl4acc0\notifications\.git\objects\pack\pack-b18ba8e6b9432e226e98554209e42a531d804693.idx
2022-12-08 22:53:02 WARNING  Removing try count: 1/3 for C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpyfl4acc0\notifications\.git\objects\pack\pack-b18ba8e6b9432e226e98554209e42a531d804693.pack
2022-12-08 22:53:02 ERROR    | notifications        | with-security        | FAIL  |    1 |
2022-12-08 22:53:02 ERROR    | notifications        | without-security     | FAIL  |    1 |
C:\Users\Administrator\.virtualenvs\opensearch-build-ppYgApDN\lib\site-packages\urllib3\connectionpool.py:1052: InsecureRequestWarning: Unverified HTTPS request is being made to host 'localhost'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
C:\Users\Administrator\.virtualenvs\opensearch-build-ppYgApDN\lib\site-packages\urllib3\connectionpool.py:1052: InsecureRequestWarning: Unverified HTTPS request is being made to host 'localhost'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
```

Notifications withour security:

<details><summary>failure log</summary>
<p>


```
2022-12-10 00:29:06 INFO     ===============================================
2022-12-10 00:29:06 INFO     Running integration tests for index-management
2022-12-10 00:29:06 INFO     ===============================================
2022-12-10 00:29:06 INFO     Executing "bash C:\Users\Administrator\opensearch-build\scripts\default\integtest.sh -b localhost -p 9200 -s true -v 2.4.1" in C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpad_5fjae\index-management
2022-12-10 01:04:02 INFO     Recording component test results for index-management at C:\Users\Administrator\opensearch-build\test-results\9fbc0d7970fe494686e28c72b503ba36\integ-test\index-management\with-security\test-results
2022-12-10 01:04:02 INFO     Integration test run failed for component index-management
2022-12-10 01:04:02 INFO     OpenJDK 64-Bit Server VM warning: Ignoring option --illegal-access=warn; support was removed in 17.0

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test add policy" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test policy successful execution" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test explain index" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test delete policy" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17


Suite: Test class org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test add policy" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:4E9BDAAF6B71977A]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.setupUsersAndRoles(IndexStateManagementSecurityBehaviorIT.kt:87)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test policy successful execution" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:A96E6FE3D053FB9C]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.setupUsersAndRoles(IndexStateManagementSecurityBehaviorIT.kt:87)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test explain index" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:C4736A6C64DD9D9F]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.setupUsersAndRoles(IndexStateManagementSecurityBehaviorIT.kt:87)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.test delete policy" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=ms-MY -Dtests.timezone=America/Jamaica -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:DD883CF2C4669CED]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT.setupUsersAndRoles(IndexStateManagementSecurityBehaviorIT.kt:87)
  2> NOTE: leaving temporary files on disk at: C:\Users\Administrator\AppData\Local\Temp\2\tmpad_5fjae\index-management\build\testrun\integTest\temp\org.opensearch.indexmanagement.IndexStateManagementSecurityBehaviorIT_7EBA5D54E4046AFB-001
  2> NOTE: test params are: codec=Asserting(Lucene94): {}, docValues:{}, maxPointsInLeafNode=237, maxMBSortInHeap=5.525916628372929, sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=ms-MY, timezone=America/Jamaica
  2> NOTE: Windows Server 2019 10.0 amd64/Eclipse Adoptium 17.0.5 (64-bit)/cpus=16,threads=1,free=340007720,total=536870912
  2> NOTE: All tests run in this JVM: [IndexManagementIndicesIT, IndexStateManagementSecurityBehaviorIT]

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test failed rollup execution user missing index access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test rollup access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test failed rollup creation user missing cluster privileges" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test delete rollup" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test rollup successful execution" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17


Suite: Test class org.opensearch.indexmanagement.RollupSecurityBehaviorIT
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test failed rollup execution user missing index access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:84B4105FA5EA800A]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.RollupSecurityBehaviorIT.setupUsersAndRoles(RollupSecurityBehaviorIT.kt:75)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test rollup access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:BD08F82DEA922AB9]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.RollupSecurityBehaviorIT.setupUsersAndRoles(RollupSecurityBehaviorIT.kt:75)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test failed rollup creation user missing cluster privileges" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:1394688E3E2320CC]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.RollupSecurityBehaviorIT.setupUsersAndRoles(RollupSecurityBehaviorIT.kt:75)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test delete rollup" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:44CCEBC3EAE2142F]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.RollupSecurityBehaviorIT.setupUsersAndRoles(RollupSecurityBehaviorIT.kt:75)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.RollupSecurityBehaviorIT.test rollup successful execution" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=it-IT -Dtests.timezone=Singapore -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:A6E458BE8BFD2D6E]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.RollupSecurityBehaviorIT.setupUsersAndRoles(RollupSecurityBehaviorIT.kt:75)
  2> NOTE: leaving temporary files on disk at: C:\Users\Administrator\AppData\Local\Temp\2\tmpad_5fjae\index-management\build\testrun\integTest\temp\org.opensearch.indexmanagement.RollupSecurityBehaviorIT_7EBA5D54E4046AFB-001
  2> NOTE: test params are: codec=Asserting(Lucene94): {}, docValues:{}, maxPointsInLeafNode=906, maxMBSortInHeap=7.677855957859007, sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=it-IT, timezone=Singapore
  2> NOTE: Windows Server 2019 10.0 amd64/Eclipse Adoptium 17.0.5 (64-bit)/cpus=16,threads=1,free=261684496,total=536870912
  2> NOTE: All tests run in this JVM: [IndexManagementIndicesIT, IndexStateManagementSecurityBehaviorIT, RollupSecurityBehaviorIT]

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.SecurityBehaviorIT.test policy access based on the backend roles" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-MX -Dtests.timezone=Asia/Yakutsk -Druntime.java=17


Suite: Test class org.opensearch.indexmanagement.SecurityBehaviorIT
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.SecurityBehaviorIT.test policy access based on the backend roles" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-MX -Dtests.timezone=Asia/Yakutsk -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:C77BAAEF2FA798A6]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.SecurityBehaviorIT.setupUsersAndRoles(SecurityBehaviorIT.kt:55)
  2> NOTE: leaving temporary files on disk at: C:\Users\Administrator\AppData\Local\Temp\2\tmpad_5fjae\index-management\build\testrun\integTest\temp\org.opensearch.indexmanagement.SecurityBehaviorIT_7EBA5D54E4046AFB-001
  2> NOTE: test params are: codec=Lucene94, sim=Asserting(RandomSimilarity(queryNorm=true): {}), locale=es-MX, timezone=Asia/Yakutsk
  2> NOTE: Windows Server 2019 10.0 amd64/Eclipse Adoptium 17.0.5 (64-bit)/cpus=16,threads=1,free=213817584,total=536870912
  2> NOTE: All tests run in this JVM: [IndexManagementIndicesIT, IndexStateManagementSecurityBehaviorIT, RollupSecurityBehaviorIT, SecurityBehaviorIT]

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test failed transform execution user missing index access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test delete transform" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test failed transform creation user missing cluster privileges" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test transform access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test stop transform" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17

REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test transform successful execution" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17


Suite: Test class org.opensearch.indexmanagement.TransformSecurityBehaviorIT
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test failed transform execution user missing index access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:7C4715A008A05CAB]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.TransformSecurityBehaviorIT.setupUsersAndRoles(TransformSecurityBehaviorIT.kt:72)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test delete transform" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:875C288602A99DA3]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.TransformSecurityBehaviorIT.setupUsersAndRoles(TransformSecurityBehaviorIT.kt:72)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test failed transform creation user missing cluster privileges" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:39D7D6E4087E063A]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.TransformSecurityBehaviorIT.setupUsersAndRoles(TransformSecurityBehaviorIT.kt:72)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test transform access" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:B6FC57399664D08C]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.TransformSecurityBehaviorIT.setupUsersAndRoles(TransformSecurityBehaviorIT.kt:72)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test stop transform" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:6A1F796F73240E2A]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.TransformSecurityBehaviorIT.setupUsersAndRoles(TransformSecurityBehaviorIT.kt:72)
  2> REPRODUCE WITH: gradlew ':integTest' --tests "org.opensearch.indexmanagement.TransformSecurityBehaviorIT.test transform successful execution" -Dtests.seed=7EBA5D54E4046AFB -Dtests.security.manager=false -Dtests.locale=es-CU -Dtests.timezone=America/Havana -Druntime.java=17
  2> org.opensearch.client.ResponseException: method [PUT], host [https://localhost:9200], URI [/_plugins/_security/api/roles/helpdesk_role], status line [HTTP/1.1 500 Internal Server Error]
    {"status":"INTERNAL_SERVER_ERROR","message":"Security index not initialized"}
        at __randomizedtesting.SeedInfo.seed([7EBA5D54E4046AFB:85B13D31F041F233]:0)
        at app//org.opensearch.client.RestClient.convertResponse(RestClient.java:375)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:345)
        at app//org.opensearch.client.RestClient.performRequest(RestClient.java:320)
        at app//org.opensearch.indexmanagement.SecurityRestTestCase.createRole(SecurityRestTestCase.kt:455)
        at app//org.opensearch.indexmanagement.TransformSecurityBehaviorIT.setupUsersAndRoles(TransformSecurityBehaviorIT.kt:72)
  2> NOTE: leaving temporary files on disk at: C:\Users\Administrator\AppData\Local\Temp\2\tmpad_5fjae\index-management\build\testrun\integTest\temp\org.opensearch.indexmanagement.TransformSecurityBehaviorIT_7EBA5D54E4046AFB-001
  2> NOTE: test params are: codec=Asserting(Lucene94): {}, docValues:{}, maxPointsInLeafNode=916, maxMBSortInHeap=7.234149462341964, sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=es-CU, timezone=America/Havana
  2> NOTE: Windows Server 2019 10.0 amd64/Eclipse Adoptium 17.0.5 (64-bit)/cpus=16,threads=1,free=428331760,total=536870912
  2> NOTE: All tests run in this JVM: [IndexManagementIndicesIT, IndexStateManagementSecurityBehaviorIT, RollupSecurityBehaviorIT, SecurityBehaviorIT, TransformSecurityBehaviorIT]

329 tests completed, 16 failed, 6 skipped

=== Standard output of node `node{::integTest-0}` ===

=== Standard error of node `node{::integTest-0}` ===

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':integTest'.
> There were failing tests. See the report at: file:///C:/Users/Administrator/AppData/Local/Temp/2/tmpad_5fjae/index-management/build/reports/tests/integTest/index.html

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 34m 55s

--- a/manifests/2.4.1/opensearch-2.4.1.yml
+++ b/manifests/2.4.1/opensearch-2.4.1.yml
@@ -176,7 +176,7 @@ components:
       - gradle:properties:version
   - name: security-analytics
     repository: https://github.com/opensearch-project/security-analytics.git
-    ref: a12f04e3c6710a0e00e367b7d2550c9127cfe721
+    ref: 2c21de6d3724402d5a33bad9f78a176bef661e07
     platforms:
       - linux
       - windows

